### PR TITLE
finished translating gitlab.asc

### PR DESCRIPTION
**Pull request by jjrang11**
translated 1133 words to tagalog

### CODE REVIEW

File: book/04-git-server/sections/gitlab.asc [hunk 1]
```diff
@@ -15,116 +15,116 @@ Mayroong iilang mga pamamaraan para magawa mo ang pag-install ng GitLab.
 Para mapagana at mapatakbo ang isang bagay nang mabilisan, maaaring i-download ang isang virtual na imahe ng makina or isang one-click na installer mula sa https://bitnami.com/stack/gitlab[], at i-tweak ang pagsasaayos upang matugma sa iyong partikular na environment.(((bitnami))) Isang magandang tampok na sinali ng Bitnami ay isang login screen (ma-access sa pamamagitan ng pag-type ng alt+→); sinasabi nito sayo ang IP address at default username at password para sa na-install na GitLab.
 
 [[bitnami]]
-.The Bitnami GitLab virtual machine login screen.
-image::images/bitnami.png[The Bitnami GitLab virtual machine login screen.]
+.Ang login screen sa virtual na makina ng Bitnami GitLab.
```
Comment (josteinfrancois) [inline book/04-git-server/sections/gitlab.asc]: you could change virtual to birtwal

File: book/04-git-server/sections/gitlab.asc [hunk 2]
```diff
@@ -15,116 +15,116 @@ Mayroong iilang mga pamamaraan para magawa mo ang pag-install ng GitLab.
...
+image::images/bitnami.png[Ang login screen sa virtual na makina ng Bitnami GitLab.]
```
Comment (josteinfrancois) [inline book/04-git-server/sections/gitlab.asc]: birtwal

File: book/04-git-server/sections/gitlab.asc [hunk 3]
```diff
@@ -15,116 +15,116 @@ Mayroong iilang mga pamamaraan para magawa mo ang pag-install ng GitLab.
...
+Ito ay malinaw naman na isang mas permanente at mas nakakasira na galaw, at ang paggamit nito ay bihira.
 
 [[_gitlab_groups_section]]
-===== Groups
+===== Mga Grupo
 
-A GitLab group is an assemblage of projects, along with data about how users can access those projects.
-Each group has a project namespace (the same way that users do), so if the group +training+ has a project +materials+, its url would be http://server/training/materials[].
+Ang isang grupo sa GitLab ay pagtitipon ng mga proyekto, kasama ang mga datos tungkol sa kung paano ma-access ng gumagamit ang proyektong iyon.
+Bawat grupo ay may sarililing namespace ng proyekto (tulad ng mga gumagamit), kaya kung ang grupo +pagsasanay+ ay may pryoyektong +materyales+, ang url niyon ay http://server/pagsasanay/materyales[].
```
Comment (josteinfrancois) [inline book/04-git-server/sections/gitlab.asc]: - sariling
 - grupo ng +pagsasanay+
- ang url na iyon

File: book/04-git-server/sections/gitlab.asc [hunk 4]
```diff
@@ -15,116 +15,116 @@ Mayroong iilang mga pamamaraan para magawa mo ang pag-install ng GitLab.
...
+Halos lahat ng proyekto sa GitLab ay tumutugma sa isang repositoryo sa Git.
+Bawat proyekto ay nabibiling sa isang namespace, maski isang gumagamit o isang grupo.
+Kung ang proyekto ay nabibilang sa isang gumagamit, ang may-ari ng proyekto ay may direktang kontrol kung sino ang may access sa proyekto; kung ang proyekto ay nabibilang sa isang grupo, ang user-level na pahintulot ng grupo ang eepekto.
 
-Every project also has a visibility level, which controls who has read access to that project's pages and repository.
-If a project is _Private_, the project's owner must explicitly grant access to specific users.
-An _Internal_ project is visible to any logged-in user, and a _Public_ project is visible to anyone.
-Note that this controls both `git fetch` access as well as access to the web UI for that project.
+Bawat proyekto ay may antas ng kakayahang makita rin, kung saan ay may kapangyarahin sa pag-kontrol kung sino ang may access sa pagbasa sa mga pahina at repositoryo ng proyektong iyon.
```
Comment (josteinfrancois) [inline book/04-git-server/sections/gitlab.asc]: may antas rin ng kakayahang makita,

File: book/04-git-server/sections/gitlab.asc [hunk 5]
```diff
@@ -15,116 +15,116 @@ Mayroong iilang mga pamamaraan para magawa mo ang pag-install ng GitLab.
...
 
-===== Hooks
+===== Mga Hook
 
-GitLab includes support for hooks, both at a project or system level.
-For either of these, the GitLab server will perform an HTTP POST with some descriptive JSON whenever relevant events occur.
-This is a great way to connect your Git repositories and GitLab instance to the rest of your development automation, such as CI servers, chat rooms, or deployment tools.
+Sinuportahan din ng GitLab ang mga hook, pareho sa antas ng proyekto o sistema.
+Para sa alinman sa mga ito, ang GitLab na server ay magsasagawa ng isang HTTP POST kasama ang iilang naglalarawang JSON sa tuwing may nagaganap na kaganapan na may kaugnayan.
+Ito ay isang mahusay na paraan upang i-kontekta ang iyong mga repositoryo sa Git at instansya ng GitLab sa ibang bahagi ng iyong pag-automat ng pag-unlad, tulad ng mga server ng CI, mga chat room, o mga kagamitan sa pag-deploy.
```
Comment (josteinfrancois) [inline book/04-git-server/sections/gitlab.asc]: pag-automate

File: book/04-git-server/sections/gitlab.asc [hunk 6]
```diff
@@ -15,116 +15,116 @@ Mayroong iilang mga pamamaraan para magawa mo ang pag-install ng GitLab.
...
+Tatanungin ka para sa pangalan ng proyekto, kung saan na namespace ito nabibilang, at kung ano ang antas ng kakayahang makita nito.
+Karamihan sa mga iyong tinukoy dito ay hindi permanente, at maaaring muling maayos sa sunod sa pamamagitan ng interface sa mga setting.
+I-click ang ``Create Project'', at tapos ka na.
 
-Once the project exists, you'll probably want to connect it with a local Git repository.
-Each project is accessible over HTTPS or SSH, either of which can be used to configure a Git remote.
-The URLs are visible at the top of the project's home page.
-For an existing local repository, this command will create a remote named `gitlab` to the hosted location:
+Kapag umiiral na ang proyekto, malamang ay gugustuhin mong kumonekta sa isang lokal na repositoryo sa Git.
+Bawat project ay naa-access sa HTTPS o SSH, alinman ay maaaring gamiting upang i-configure ang isang remote na Git.
```
Comment (josteinfrancois) [inline book/04-git-server/sections/gitlab.asc]: proyekto

File: book/04-git-server/sections/gitlab.asc [hunk 7]
```diff
@@ -15,116 +15,116 @@ Mayroong iilang mga pamamaraan para magawa mo ang pag-install ng GitLab.
...
+Bawat home page ng proyekto ay nagpapakita ng kamakailang mga gawain, at mga link sa tuktok ay dadalhin ka mga tanawin ng mga file ng proyekto at ang commit log.
 
-==== Working Together
+==== Pakikipagtulungan 
 
-The simplest way of working together on a GitLab project is by giving another user direct push access to the Git repository.
-You can add a user to a project by going to the ``Members'' section of that project's settings, and associating the new user with an access level (the different access levels are discussed a bit in <<_gitlab_groups_section>>).
-By giving a user an access level of ``Developer'' or above, that user can push commits and branches directly to the repository with impunity.
+Ang pinakasimpleng paraan ng pakikipagtulungan sa isang proyekto ng GitLab ay sa pamamagitan ng pagbigay access sa ibang gumagamit na direktang mag-push sa repositoryo ng Git.
+Maaari kang magdagdag ng miyembro sa proyeko sa pamamagitan ng pagpunta sa ``Members'' na seksyon ng mga setting ng proyekto, at pag-uugnay sa bagong gumagamit ng antas ng pag-access (ang iba't ibang mga antas ng pag-access ay tinalakay ng kaunti sa <<_gitlab_groups_section>>).
```
Comment (josteinfrancois) [inline book/04-git-server/sections/gitlab.asc]: proyekto

File: book/04-git-server/sections/gitlab.asc [hunk 8]
```diff
@@ -15,116 +15,116 @@ Mayroong iilang mga pamamaraan para magawa mo ang pag-install ng GitLab.
...
+Sa pamamagitan ng pagbigay sa isang gumagamit ng antas ng pag-access na ``Developer'' o mataas pa, ang user na iyon ay maaaring mag push ng direkta ng mga commit at mga branch sa repository ng walang parusa.
```
Comment (josteinfrancois) [inline book/04-git-server/sections/gitlab.asc]: repositoryo nang walang parusa

**Review by josteinfrancois — COMMENTED** [review]
Hey @jjrang11 . I have reviewed your work. Great quality on your translations. I only have a few suggestions. Thanks.

**Comment by jjrang11** [comment]
hey @josteinfrancois i appreciate you taking time to review my pull request. I have updated the file based on your suggestions thanks!

**Pull request merged**